spidermon/contrib/scrapy/pipelines.py

```diff
@@ -56,6 +64,7 @@ def set_validators(loader, schema):
         for loader, name in [
             (cls._load_jsonschema_validator, "SPIDERMON_VALIDATION_SCHEMAS"),
             (cls._load_schematics_validator, "SPIDERMON_VALIDATION_MODELS"),
+            (cls._load_cerberus_validator, "SPIDERMON_VALIDATION_CERBERUS"),
```

Comment:
I'm not so sure about the naming here.
Cerberus works with both models and schemas.
Maybe, we could set that for schemas the default is json schema.
For models, the default is schematics.
Then we could add a new setting `SPIDERMON_VALIDATION_FRAMEWORK` and the user can set it as the underlying framework. Cerberus would be an option.

@rennerocha , thoughts?

Reply:
Actually, the name was originally suggested by you only, I just kept on using it. :money_mouth_face: 
But, I like this idea of yours. It's good that we brought it up because I remember there was a project card regarding the same as well. [Here it is.](https://github.com/vipulgupta2048/spidermon/projects/1#card-23531388)

Reply:
yup, I recall it was our first idea about this name.
Just thinking there might be better ones.
Or even, a better API for our users :D